drivers/gpio/Kconfig.pca_series

```diff
@@ -0,0 +1,43 @@
+# PCA_SERIES GPIO configuration options
+
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_PCA_SERIES
+	bool "PCA_SERIES I2C-based GPIO chip"
+	default y
+	depends on DT_HAS_NXP_PCA9538_ENABLED || DT_HAS_NXP_PCA9539_ENABLED || \
+		   DT_HAS_NXP_PCA9574_ENABLED || DT_HAS_NXP_PCA9575_ENABLED || \
+		   DT_HAS_NXP_PCAL9538_ENABLED || DT_HAS_NXP_PCAL9539_ENABLED || \
+		   DT_HAS_NXP_PCAL6524_ENABLED || DT_HAS_NXP_PCAL6534_ENABLED
+	depends on I2C
+	help
+	  Enable driver for PCA_SERIES I2C-based GPIO chip.
+
+# TODO: add this line
```

Comment:
add back this line when it is ready to retire pcal64xxa driver. currently they are conflict

Reply:
@danieldegrasse Is it better if I remove support for PCAL6408A and PCAL6416A in this pr, and open another pr to add back the support? Since retiring these drivers will need to edit a bunch of dts from other vendor...